lib/local-links-manager/import/services_tier_importer.rb

```diff
@@ -41,34 +41,68 @@ def import_item(item, response, summariser)
         service = Service.find_by(lgsl_code: item[:lgsl_code])
         raise MissingRecordError, "LGSL #{item[:lgsl_code]} is missing" if service.nil?
         Rails.logger.info("Updating service '#{service.label}' (lgsl #{service.lgsl_code})")
+        checked_services.add(service)
 
         if item[:tier].blank?
           response.errors << "LGSL #{item[:lgsl_code]} is missing a tier"
           summariser.increment_ignored_items_count
-        elsif not update_tier(service, item[:tier])
+        elsif !service.valid_tier?(item[:tier])
           response.errors << "LGSL #{item[:lgsl_code]} has incorrect tier name"
           summariser.increment_ignored_items_count
         else
+          service.delete_and_create_tiers(item[:tier])
```

Comment:
If this is run for every service, then all the `created_at` timestamps will be the same, right?